lib/telemetry_metrics.ex

```diff
@@ -77,18 +77,38 @@ defmodule Telemetry.Metrics do
 
   Metric specifications are created using one of the four functions: `counter/2`, `sum/2`,
   `last_value/2` and `distribution/2`. Each of those functions returns a specification of metric
-  of the corresponding type. The first argument to all these functions is the name of events which
+  of the corresponding type.
+
+  The first argument to all these functions is the metric source. Metric source may take three
+  different shapes:
```

Comment:
Please add another new line after `:` and indent every bullet below with two spaces. :)